### PR TITLE
Add a small example for influence computation to README.md

### LINKED ISSUE #335: Add influences usage example to README
To show that we also have support for influences we should add a short usage example to the readme just like we have one for data valuation.

### PR DESCRIPTION
**Pull request by schroedk**
### Description

This PR closes #335 

### Changes

- add a small example for influence computation to README.md

### Checklist

- ~~[ ] Wrote Unit tests (if necessary)~~
- [x] Updated Documentation (if necessary)
- ~~[ ] Updated Changelog~~
- ~~[ ] If notebooks were added/changed, added boilerplate cells are tagged with `"nbsphinx":"hidden"`~~

### CODE REVIEW

**Review by Xuzzo — APPROVED** [review]
Looks good

**Comment by mdbenito** [comment]
Shouldn't we explicitly mention that Arnoldi is better than CG and why? I understand it makes no sense for this example but it might be better to have it on "the front page" so to speak.

File: README.md [hunk 1]
```diff
@@ -99,6 +99,55 @@ documentation.
...
+2. Compute influence factors by providing training data and inversion method
+
+Using the conjugate gradient algorithm, this would look like:
+```python
+import torch
+from torch import nn
+from torch.utils.data import DataLoader, TensorDataset
+
+from pydvl.influence import TorchTwiceDifferentiable, compute_influences
+from pydvl.influence.inversion import InversionMethod
```
Comment (mdbenito) [inline README.md:116]: I would add InversionMethod to `influence/__init__.py` for convenience while we are at it.

File: README.md [hunk 2]
```diff
@@ -99,6 +99,55 @@ documentation.
...
+
+nn_architecture = nn.Sequential(
+    nn.Conv2d(in_channels=5, out_channels=3, kernel_size=3),
+    nn.Flatten(),
+    nn.Linear(27, 3),
+)
+loss = nn.MSELoss()
+influence_model = TorchTwiceDifferentiable(nn_architecture, loss)
```
Comment (mdbenito) [inline README.md:124]: Sorry for nitpicking, but isn't `influence_model` a misnomer? One is computing influences for the model, not a model of influences. I would just leave it as `influence`.
```suggestion
model = TorchTwiceDifferentiable(nn_architecture, loss)
```

File: README.md [hunk 3]
```diff
@@ -99,6 +99,55 @@ documentation.
...
+    TensorDataset(torch.rand((10, *input_dim)), torch.rand((10, output_dim))),
+    batch_size=2,
+)
+test_data_loader = DataLoader(
+    TensorDataset(torch.rand((5, *input_dim)), torch.rand((5, output_dim))),
+    batch_size=1,
+)
+
+influences = compute_influences(
+    influence_model,
```
Comment (mdbenito) [inline README.md:139]:
```suggestion
    model,
```

Comment (schroedk) [reply README.md:139]: Done

File: README.md (hunk 2 above)
Comment (schroedk) [reply README.md:124]: Done

File: README.md (hunk 1 above)
Comment (schroedk) [reply README.md:116]: Done
